src/pyTOPP/src/pyopenms/pytopp/tools/pytopp_example_cli.py

```diff
@@ -1,10 +1,19 @@
+#!/usr/bin/env -S uv run --script
+# /// script
+# requires-python = ">=3.9"
+# dependencies = [
+#   "pyopenms>=3",
+#   "CTDopts @ git+https://github.com/WorkflowConversion/CTDopts.git",
```

Comment:
@jpfeuffer would it make sense to move the single file of CTDopts into pyTOPP? It is not maintained anymore (was developed by a colleague) and we might need to update it at some point.